### PR TITLE
GovUK Frontend V5

### PR DESCRIPTION
**Pull request by psoleckimoj**

### CODE REVIEW

File: server/middleware/setUpWebSecurity.ts [hunk 1]
```diff
@@ -7,15 +8,35 @@ export default function setUpWebSecurity(): Router {
...
+          scriptSrc: [
+            "'self'",
+            'www.gstatic.com',
+            'code.jquery.com',
+            (_req: Request, res: Response) => `'nonce-${res.locals.cspNonce}'`,
+          ],
+          styleSrc: [
+            "'self'",
+            'www.gstatic.com',
+            'code.jquery.com',
```
Comment (tpmcgowan) [inline server/middleware/setUpWebSecurity.ts:35]: is jQuery domain needed if you've removed it in `layout.njk` and using it locally? Might be used by MoJ library I guess 🤷‍♂️

**Review by tpmcgowan — APPROVED** [review]
LGTM - just minor query about CSP domains

File: server/middleware/setUpWebSecurity.ts (hunk 1 above)
Comment (psoleckimoj) [reply server/middleware/setUpWebSecurity.ts:35]: I'll be honest, I'm not sure, I just pulled across from the template app. I'll check in a bit.

Comment (psoleckimoj) [reply server/middleware/setUpWebSecurity.ts]: @tpmcgowan I've removed jquery completely. I wonder if it's even needed in the template 🤔

Comment (tpmcgowan) [reply server/middleware/setUpWebSecurity.ts]: MoJ Pattern library says it needs jQuery (https://design-patterns.service.justice.gov.uk/get-started/setting-up-javascript/) but I guess that might be dependent on what components you're using. But the template has jQuery and jQuery UI in its `layout.njk` 🤔 .

**Review by tpmcgowan — APPROVED** [review]
LGTM

File: server/middleware/setUpWebSecurity.ts (hunk 1 above)
Comment (psoleckimoj) [reply server/middleware/setUpWebSecurity.ts]: Good spot. We don't actually use any moj-frontend components in this app. I'll see about a PR to remove.

**Pull request merged**